app/assets/stylesheets/common.css.scss

```diff
@@ -0,0 +1,23 @@
+.center { text-align: center }
```

Comment:
Name of this file will add Deprecation warnings in log, better to rename it to `common.scss`.